wade/chorus.py

```diff
@@ -30,7 +30,10 @@
 TIMEOUT = 'timeout'
 RECV_BYTES = 1024 * 64
 SEND_BYTES = 1024 * 64
-OUTGOING_PEER_BUFFER_SIZE = 1024 * 64
+
+OUTGOING_PEER_BUFFER_SIZE = 1024 ** 2
+"""1 MB, this controls the max size of a request that can be sent to WADE."""
```

Comment:
nit: I'm not a fan of the docstring after the constant. I know this appears in Chartbeat code, but I don't see it elsewhere much. No other language does this as far as I know.